scripts-local/vars.sh

```diff
@@ -25,6 +25,8 @@ DAY_EPOCH_LEN="100s"
 STRIDE_EPOCH_INDEX=2
 STRIDE_EPOCH_LEN="40s"
 IBC_TX_WAIT_SECONDS=30
+MAX_DEPOSIT_PERIOD="30s"
```

Comment:
Why 30s instead of 60m?

Reply:
This is just the initial setup so that you run locally, submit a proposal, and test the upgrade. These settings are generalized across all local mode upgrade runs - they are not connected to the actual upgrade logic (which in this case happened to also be updating these parameters)